### PR TITLE
Add PR labeler action to label translation-related PRs

### PR DESCRIPTION
**Pull request by mingyc**

### CODE REVIEW

File: .github/labeler.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+- po/bn.po
+
+'translation/da':
+- po/da.po
+
+'translation/de':
+- po/de.po
+
+'translation/el':
+- po/el.po
```
Comment (mgeisler) [inline .github/labeler.yml:14]: Nice idea! I'm worried about having to maintain the list of languages here, but I guess we can add them on a best-effort basis?

**Review by mgeisler — APPROVED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (mingyc) [reply .github/labeler.yml:14]: Yes. [This action](https://github.com/actions/labeler#inputs) does not support programmatically adding labels. The configuration here falls back to `translation` label by default for new languages.

Comment (mgeisler) [reply .github/labeler.yml:14]: Okay, sounds good. Could you add a line to `TRANSLATIONS.md` to ask people to update the configuration when adding a new translation?

Comment (mingyc) [reply .github/labeler.yml:14]: Done.

Comment (mgeisler) [reply .github/labeler.yml:14]: Thank you!

**Pull request merged**